lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -16,14 +16,14 @@ class InstructionSlide extends React.Component {
           <div className="wrapper">
 
             <div className="container__row">
-              <div className="container__block">
-                <h2 className="heading">This is how you do it.</h2>
+              <div className="container__block -narrow -centered">
+                <h2 className="heading -gamma">This is how you do it.</h2>
                 <p>We are PUMPED to see your photos. Complete these simple steps and upload a photo of yourself in action. Your pics will inspire others to join the movement &ndash; so don't forget to share them!</p>
               </div>
```

Comment:
Pulling the copy out into variables would fix the syntax highlighting issue, and might make things a bit nicer to read for larger blocks of text like this.

```js
const heading = `This is how you do it.`;
const body = `We are PUMPED to see your photos. Complete these simple steps and uploa...`;

return (
  <div className="container__block -narrow -centered">
    <h2 className="heading -gamma">{heading}</h2>
    <p>{body}</p>
```